src/map/battle.cpp

```diff
@@ -411,7 +420,12 @@ int battle_delay_damage(t_tick tick, int amotion, struct block_list *src, struct
 	dat->additional_effects = additional_effects;
 	dat->src_type = src->type;
 	dat->isspdamage = isspdamage;
-	if (src->type != BL_PC && amotion > 1000)
+	if (battle_config.synchronize_damage && skill_id == 0 && src->type == BL_MOB && amotion > status_get_clientamotion(src)) {
+		// The client refuses to display animations slower than 1x speed
+		// So we need to shorten AttackMotion to be in-sync with the client in this case
+		amotion = status_get_clientamotion(src);
+	}
+	else if (src->type != BL_PC && amotion > 1000)
```

Comment:
```suggestion
	if (src->type != BL_PC && amotion > 1000)
```

Reply:
We need to put the if(battle_config.synchronize_damage [...]" part here because otherwise you'd cap amotion to 1000 even if the feature applies.

Reply:
Everything above 432 (so 1000) will be ignored by the client as you commented.
So everything is fine.

Reply:
I think you are missing that the amotion is something completely different from the value we want to send to the client.

Amotion is the time in milliseconds until which the damage shows (here it's only used to create the timer). This isn't limited to 432.

What is (technically) limited to 432 is the what we send to the client, but this is not a value in milliseconds but rather the inverted animation speed.

One monster where this is relevant would be Medusa:
```
    AttackMotion: 1320
    ClientAttackMotion: 1080
```
Here we want amotion in battle.cpp to be 1080.

The value we send to the client would be 432 (or higher), so it plays at 1x speed which results in 1080ms until the damage shows.

Reply:
Actually after sleeping about it, why don't we move the 1000 limit also to the top? Then we can simplify the code as you suggested without breaking the feature.

If I look at the code "amotion" is not used anywhere except the <=1 check and for the timer at the end.

Reply:
Done!